### PR TITLE
Fix OCI artifact generation

### PR DESCRIPTION
**Pull request by IgorTodorovskiIBM**

### CODE REVIEW

File: cicd/publish_oci.sh [hunk 1]
```diff
@@ -52,8 +52,24 @@ LABEL vendor=\"ZOSOpenTools\""
...
+  exit 1
+fi
+
+set -e
+cd "\$INSTALL_DIR/${DIR_NAME}"
+./setup.sh
+ZZ
+
+  cat << ZZ > .zpm/.env
+cd "\$INSTALL_DIR/${DIR_NAME}";
```
Comment (MikeFultonDev) [inline cicd/publish_oci.sh:69]: do you want to protect this better? What if the .env did a cd as well?

Comment (MikeFultonDev) [reply cicd/publish_oci.sh:69]: never mind... i mis-read and thought it was . ./setup.sh

**Review by MikeFultonDev — APPROVED** [review]
LGTM

**Comment by IgorTodorovskiIBM** [comment]
Added runtime dependencies as a label. Once the spec if official, we will modify accordingly

**Pull request merged**